### PR TITLE
Create asdf.ghj

### PR DESCRIPTION
**Pull request by derekoverby**
you cant write code without qwerty

### CODE REVIEW

**Comment by bridevauxj** [comment]
your avatar is great

File: asdf.ghj [hunk 1]
```diff
@@ -0,0 +1,16 @@
+a
+b
+c
+d
+e
+f
+g
+h
+i
+j
```
Comment (bridevauxj) [inline asdf.ghj]: This letter starts the best names.

**Comment by bridevauxj** [comment]
@derekoverby remove the letter c because s and k already make those sounds.

**Comment by derekoverby** [comment]
@bridevauxj roger, roger

**Comment by derekoverby** [comment]
@bridevauxj good to go?

**Comment by bridevauxj** [comment]
:+1:  you get a doughnut :doughnut:

**Pull request merged**